rasa/utils/plotting.py

```diff
@@ -124,6 +125,8 @@ def plot_confusion_matrix(
         logger.info(f"Confusion matrix, without normalization: \n{confusion_matrix}")
 
     thresh = zmax / 2.0
+    if not thresh:
+        raise RasaException(f"The variable `thresh` cannot be `{type(thresh)}`.")
```

Comment:
we should in my opinion also not raise `RasaException` directly but some more specific sub type of it

Reply:
True! Maybe I can raise a `TypeError` like [here](https://github.com/RasaHQ/rasa/blob/1aa1b6c93ba2603064737814f9fcdce764af40d9/rasa/core/actions/action.py#L799)

Reply:
I don't think `TypeError` is ignored by Sentry 🤔 what about adding a new RasaException subclass for this [here](https://github.com/RasaHQ/rasa/blob/1aa1b6c93ba2603064737814f9fcdce764af40d9/rasa/exceptions.py)?